frontend/src/utilsStorage.ts

```diff
@@ -0,0 +1,61 @@
+export enum StorageKeys {
+  // Active Page Keys
+  Active_SessionStorage_View = 'ActivePageView',
+  Active_SessionStorage_Header = 'ActivePageHeaders',
+  Active_SessionStorage_ID_WS = 'ActivePageIDWorkspace',
+  Active_SessionStorage_ID_T = 'ActivePageIDTemplate',
+  // Dashboard Keys
+  Dashboard_SessionStorage_View = 'DashboardPageView',
+  Dashboard_SessionStorage_ID_T = 'DashboardPageTemplate',
```

Comment:
The `SessionStorage` word is not necessary. Actually you could decide to store the same value both in session and local storage. It is decided based on what implementation you chose when instantiating the StorageValue.
You could directly put the key of the enum equal to the value